app/src/release/res/values/manifest.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <!-- This is the SHA-256 digest for the Authenticator App Release variant:-->
+    <!-- This is the SHA-256 digest for Google Play signing key of theAuthenticator App Release
```

Comment:
`theAuthenticator ` - > `the Authenticator`

Reply:
46e75585e68aba9ed633f25e16aae9503ccbde58